service/src/integrationTest/kotlin/fi/espoo/evaka/assistanceneed/AssistanceNeedVoucerCoefficientEndOutdatedTest.kt

```diff
@@ -124,7 +138,12 @@ class AssistanceNeedVoucerCoefficientEndOutdatedTest : PureJdbiTest(resetDbBefor
         }
         assertEquals(listOf(FiniteDateRange(jan1, dec31)), getCoefficientRanges())
 
-        db.transaction { tx -> tx.endOutdatedAssistanceNeedVoucherCoefficients(today) }
+        db.transaction { tx ->
+            tx.endOutdatedAssistanceNeedVoucherCoefficients(
+                user = AuthenticatedUser.SystemInternalUser,
+                now = clock.now(),
+            )
+        }
 
         assertEquals(listOf(FiniteDateRange(jan1, yesterday)), getCoefficientRanges())
```

Comment:
I would like to see an assertion for the changed modified metadata here. You could do this by modifying the `getCoefficientRanges` into:
```
getCoefficients() =
        db.read {
            it.createQuery {
                    sql(
                        """
    SELECT coe.*, evaka_user.name AS modifiedByName
    FROM assistance_need_voucher_coefficient AS coe LEFT JOIN evaka_user on coe.modified_by = evaka_user.id
    ORDER BY validity_period
    """
                    )
                }
                .toList<AssistanceNeedVoucherCoefficient>()
        }
}
```
and then check that modified* fields have correct values:
```


        val coefficientsAfterEndOutdated = getCoefficients()
        println(coefficientsAfterEndOutdated)
        assertEquals(
            listOf(FiniteDateRange(jan1, yesterday)),
            coefficientsAfterEndOutdated.map { it.validityPeriod },
        )
        assertEquals(
            listOf(clock.now()),
            coefficientsAfterEndOutdated.map { it.modifiedAt },
        )
        assertEquals(
            listOf("eVaka"),
            coefficientsAfterEndOutdated.map { it.modifiedByName },
        )
        ```